tools/ci/setup_vmdb_configs.sh

```diff
@@ -0,0 +1,6 @@
+echo "1" > REGION
+cp certs/v2_key.dev certs/v2_key
+cp config/database.pg.yml config/database.yml
+cp config/cable.yml.sample config/cable.yml
+psql -c "CREATE USER root SUPERUSER PASSWORD 'smartvm';" -U postgres
+
```

Comment:
Looks good @durandom 👍 

If these changes work, it's good to bucket these various setups into categories.